### DISCUSSION TITLE
Is it possible to grant the mobile APP permission to SAML/SSO organisations?

### DISCUSSION THREAD

**Discussion by tom-sherman** [community/community#71] [Mobile] [answered]
I want to view notifications, PRs etc. from my work's organisation. When I open a link in that organisation in the app I get a "Resource protected by organization SAML enforcement. You must grant your OAth token access to this organisation". How do I do this?

EDIT: Ooooo, turns out I have to logout and log back in again? It would be good if I didn't need to do that after being added to an organisation.

**Comment by candyho** [comment, marked as answer]
Hey thanks for sharing your feedback. This is a known issue and apologise for the inconvenience caused. The current way to gain access to all the protected orgs is to log out and log back in again. We have an issue in our internal repo to track this and we will keep you posted once we land on a solution.

**Reply by tom-sherman** [reply]
@candyho Thanks! Slightly aside, I got this weird user in my notifications for this reply, I don't know if it's an error or not.

![image](https://user-images.githubusercontent.com/9257001/99685459-83c47880-2a7a-11eb-99ba-121e64244854.png)

**Reply by candyho** [reply]
Ohh thanks for flagging. It does look like a bug. Will check with our internal teams to investigate.

**Reply by philip-gai** [reply]
@candyho Any updates on this issue? I ran into the same thing recently.

**Reply by itsthejoker** [reply]
This is still an issue 😬

**Reply by prodigysml** [reply]
Can confirm, this is still an issue... Please prioritise this as it's a major pain.

**Reply by emredoganm** [reply]
This is still an issue 😅

**Reply by andrei-hernandez** [reply]
This is still an issue

**Reply by candyho** [reply]
Thanks, everyone, for your patience! I’m excited to share that this has already been fixed on Android and is currently being worked on for iOS. We appreciate your understanding as we work to complete the iOS update, so please bear with us just a little longer.

**Comment by isabellamarie12** [comment]
Yes, it’s definitely possible to grant mobile app permissions to SAML/SSO organizations. In fact, implementing SAML/SSO for mobile apps can streamline the authentication process, making it easier for users to securely access their accounts with a single set of credentials. This not only enhances security but also improves the overall user experience by reducing the need for multiple logins. It’s all about setting up the right integration to ensure the app communicates smoothly with the identity provider. If done right, it can really simplify things for both users and admins.

**Reply by ebndev** [reply]
Hi @isabellamarie12, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by qMatr1xp** [comment]
gh.io/aup.
https://github.com/subprocessors
